polaris-service/src/main/java/org/apache/polaris/service/PolarisApplication.java

```diff
@@ -288,7 +289,7 @@ public void run(PolarisApplicationConfig configuration, Environment environment)
     // different processes
     // and in-memory state will be lost b/w invocation of bootstrap command and running a server
     if (metaStoreManagerFactory instanceof InMemoryPolarisMetaStoreManagerFactory) {
-      metaStoreManagerFactory.getOrCreateMetaStoreManager(configuration::getDefaultRealm);
+      metaStoreManagerFactory.bootstrapRealms(List.of(configuration.getDefaultRealm()));
     }
```

Comment:
Can we move this logic into class `InMemoryPolarisMetaStoreManagerFactory` as a part of initialization? Actually it is already there, but deleted in this PR. IIUC, here is the logic. We can make this change due to `getOrCreateMetaStoreManager()` is invoked already before this line. In that case, why do we need to bootstrap again here? We can bootstrap in the method `InMemoryPolarisMetaStoreManagerFactory::getOrCreateMetaStoreManager`

Reply:
I want to make the behavior the same regardless of InMemory or EclipseLink so we can test against both with the same set of tests. We will not bootstrap automatically but bootstrap if needed - we bootstrap in the tests for both in-memory and EclipseLink and here in service against in-memory.

Reply:
I understand that we want to explicitly bootstrap in test, which applies to both in-memory and eclipse-link. But this place is a bit different. I was wondering if there is a better way to handle bootstrapping here. Ideally, it should be an internal operation within `InMemoryPolarisMetaStoreManagerFactory`.

Reply:
I reverted the change actually since the users actually rely on the output to get the credentials from console.